src/components/virtualRepeat/virtual-repeater.js

```diff
@@ -100,6 +102,25 @@ function VirtualRepeatContainerController($$rAF, $scope, $element, $attrs) {
   /** @type {number} Amount to offset the total scroll size by. */
   this.offsetSize = parseInt(this.$attrs.mdOffsetSize, 10) || 0;
 
+  if(this.$attrs.hasOwnProperty('mdTopIndex')) {
```

Comment:
By the way I found an issue where the app crashes if `md-top-index=""`. So I am changing this line to `if (this.$attrs.mdTopIndex) {` to make sure it is defined and specified.